### PR TITLE
Sinaomrani patch 1: three line menu in mobile devices

### PR DESCRIPTION
**Pull request by huzwares**
add three line menu in mobile devices

### CODE REVIEW

File: layouts/css/page-modules/_header.styl [hunk 1]
```diff
@@ -74,3 +74,125 @@ header
     border none
     max-height 75px
     max-width 180px
+
+  #hamburger
+    display none
+
+  #Mobile ul
```
Comment (fhemberger) [inline layouts/css/page-modules/_header.styl:81]: please use lowercase IDs.

**Review by fhemberger — CHANGES_REQUESTED** [review]

**Comment by fhemberger** [comment]
This doesn't seem to work in Chrome/Mac for me:

bildschirmfoto 2017-02-10 um 20 24 23

File: layouts/css/page-modules/_header.styl [hunk 2]
```diff
@@ -74,3 +74,125 @@ header
...
+  #Mobile ul li a 
+    font-size 1.1em
+
+  #Mobile #logo img 
+    margin .5em auto .5em auto
+
+  .menu 
+    text-align center
+    display none
+    background-color #333333
```
Comment (hiroppy) [inline layouts/css/page-modules/_header.styl:97]: Please use this(https://github.com/nodejs/nodejs.org/blob/dd12f6d/layouts/css/_variables.styl#L18).

**Review by hiroppy — CHANGES_REQUESTED** [review]

**Comment by fhemberger** [comment]
@sinaomrani Hi, can you please check the comments/requested changes?

**Comment by fhemberger** [comment]
As the patch doesn't work and the committer doesn't seem to work on this any longer, I'm closing this. @sinaomrani Please feel free to open a new version of this PR, which includes the requested changes.

**Pull request closed without merging**